sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -238,13 +238,13 @@ class Analyzer(
       ResolveNaturalAndUsingJoin ::
       ResolveOutputRelation ::
       ExtractWindowExpressions ::
+      ResolveTimeZone(conf) ::
```

Comment:
This change will be reverted after #28288 merged.

Reply:
ok, merged in https://github.com/apache/spark/commit/ca90e1932dcdc43748297c627ec857b6ea97dff7

Reply:
Thanks for the help! Reverted in c75fe08